src/rust/ensogl/lib/components/src/toggle_button.rs

```diff
@@ -64,6 +64,94 @@ impl<Shape:ColorableShape+'static> Model<Shape> {
 
 
 
+// ===================
+// === ButtonState ===
+// ===================
+
+/// A state a button can be in.
+#[derive(Clone,Copy,Debug,Default)]
```

Comment:
Should the default for a button be visible instead of not visible?

Reply:
Of course it should, thansk for cathcing it